shared/actions/favorite.js

```diff
@@ -165,7 +165,7 @@ const jsonToFolders = (json: Object, myKID: any) => {
 export function favoriteList (): (dispatch: Dispatch, getState: () => Object) => void {
   return (dispatch, getState) => {
     // Ignore any messages we receive while reloading
-    if (getState().dev.reloading) {
+    if (getState().dev && getState().dev.reloading) {
```

Comment:
we don't want a crash having to do with this dev functionality, this is just for HMR so...